tests/lib/qunit/addons/composite/qunit-composite.js

```diff
@@ -39,6 +39,10 @@ function runSuite( suite ) {
 	}
 
 	QUnit.asyncTest( suite, function() {
+		// See: https://github.com/JamesMGreene/qunit-composite/issues/20
```

Comment:
It appears that this is no longer needed, at least not in single runs with Vagrant (Firefox and Chrome tested) or on my machine (Safari, Opera, Chrome, Firefox, and PhantomJS tested).  I have removed that workaround, if it comes up again we have earlier commits and this conversation to refer to.